python/datashaper/datashaper/workflow.py

```diff
@@ -352,10 +354,14 @@ def run(
                 result = asyncio.run(result)
 
             # Record Verb Timing
-            if on_verb_timing is not None:
-                on_verb_timing(
-                    f"verb_{verb_name}_{verb_idx}", time.time() - start_verb_time
+            verb_timings.append(

```

Comment:
are we always returning the timings? could be part of the @verb decorator and make the verb return timing, result